impl/src/test/java/org/ehcache/impl/internal/store/tiering/CacheStoreTest.java

```diff
@@ -680,6 +683,36 @@ public void testReleaseStoreFlushes () throws CacheAccessException {
     verify(cachingTierProvider, times(1)).releaseCachingTier(any(CachingTier.class));
   }
 
+  @Test
+  public void testRank() throws Exception {
+    CacheStore.Provider provider = new CacheStore.Provider();
+
+    Assert.assertThat(provider.rank(
+        Collections.<ResourceType>singleton(ResourceType.Core.DISK), Collections.<ServiceConfiguration<?>>emptyList()),
+        Matchers.is(0));
```

Comment:
More style than anything, but using static imports here helps readability for me.